proxy/interceptor_test.go

```diff
@@ -27,6 +27,8 @@ package proxy
 import (
 	"context"
 	"fmt"
+	"go.temporal.io/api/export/v1"
+	"go.temporal.io/api/history/v1"
```

Comment:
Go fmt I would expect would fix this to move non-stdlib imports to later, but no biggie